inc/dps/dps.h

```diff
@@ -776,6 +776,17 @@ DPS_Status DPS_InitPublication(DPS_Publication* pub,
                                const DPS_KeyId* keyId,
                                DPS_AcknowledgementHandler handler);
 
+/**
+ * Configure the Quality-of-Service (QoS) parameters of a publication.
```

Comment:
Maybe define a QoS struct that can be extended if new QoS capabilities are added in the future?

Reply:
Yes, I'll do this